app/controllers/invitations_controller.rb

```diff
@@ -36,6 +36,17 @@ def resource_params
 
   # Overwrite Devise::Invitations resource_class method
   def resource_class
-    params.dig(:invitation, :type).constantize
+    return params.dig(:invitation, :type).constantize if params.dig(:invitation, :type).present?
+
+    User
```

Comment:
> Could this be super instead of being explicitly User?

Yes, that's what I wanted to do, but super will return an UnauthorizedUser class instead of User because of the route I created. Let me see if I can fix it.

With a little more work we could remove the need for this method altogether. But that will also require frontend updates as well, which at the time this was created I wasn't willing to do, but I am now.

Ideally we can get some tests written to cover the current correct behavior, and then make the updates to remove this method. I will get an issue created after this is merged, and maybe someone will grab it.